src/libs/ErrorUtils.js

```diff
@@ -54,8 +54,30 @@ function getLatestErrorMessage(onyxData) {
         .value();
 }
 
+/**
+     *
+     * @param {Object} errors - An object containing current errors in the form
+     * @param {String} key - inputID of the field
```

Comment:
If we call this `inputID` then we don't have to explain that it's the inputID :trollface: